src/fullnessTask.cpp

```diff
@@ -27,13 +34,46 @@ void FullnessTask::taskFunction(void *task)
 
 void FullnessTask::setup()
 {
+    Serial.begin(115200);
+    pinMode(trigPin, OUTPUT);
+    pinMode(echoPin, INPUT);
 }
 
 void FullnessTask::loop()
 {
     while (1)
     {
-        vTaskDelay(1000 / portTICK_PERIOD_MS); // Delay for 1000 milliseconds
-        log_i("Hello from Fullness Task");
+        if (amount < 8)
+        {
+            digitalWrite(trigPin, LOW);
+            vTaskDelay(1000 / portTICK_PERIOD_MS); // Delay for 1000 milliseconds
+            digitalWrite(trigPin, HIGH);
+            vTaskDelay(1000 / portTICK_PERIOD_MS); // Delay for 1000 milliseconds
+            digitalWrite(trigPin, LOW);
```

Comment:
Add all of the library code to a new class called `UltrasonicDistance` that inherits from IDistance